src/core/ucc_lib.c

```diff
@@ -4,10 +4,9 @@
  * See file LICENSE for terms.
  */
 
-#ifdef HAVE_CONFIG_H
-#  include "config.h"
-#endif
+#include "config.h"
 
+#include "ucc_global_opts.h"
```

Comment:
@Sergei-Lebedev  what do you mean? I was thinking that we always use quotes for internal headers and <> for external ones. What is our style ?

Reply:
I'm asking because we have both quotes and <> here
```C
#include "ucc_global_opts.h"
#include <api/ucc.h>
```
or we suppose api/ucc.h is external?

Reply:
fixed